README.md

```diff
@@ -1,75 +1,88 @@
+(https://drive.google.com/file/d/1yPxjEHKIb629FyoHiB8xIgkUQjGIIotT/view?usp=sharing)
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
+## Getting Started
+
+If you want to clone this project:
+- Go to GitHub repository
+- Click on the green button that say 'Code'
+- Copy the SSH link
+- On your local machine, type the command git clone and paste the SSH link
 
-To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+- Code editor
+- Knowledge in HTML
+- Knowledge in CSS
+- Knowledge in JavaScript
+- Knowledge in React
+- Knowledge in Redux
+- Handle on local repository Git
+
 
 ### Setup
+- Clone the repository
+- Change directory into the project folder
+- Install dependencies
+- Open project with text editor to view changes
+- Run the code using [live server](https://www.google.com/search?client=safari&rls=en&q=live+server&ie=UTF-8&oe=UTF-8)
 
 ### Install
+Clone this repository (you must have [Git Installed](https://github.com/git-guides/install-git))
 
-### Usage
+`git clone git@github.com:robinbenitezmora/crypto-coins.git`
 
-### Run tests
+Then, move to the newly created folder `crypto-coins` using this command
 
-### Deployment
+`cd crypto-coins`
 
+Then, install dependencies using this command
 
+Then, install dependencies using this command
 
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+To open the project in Visual Studio Code type the following command inside the _books_with_es6_ folder
 
-👤 **Author2**
+`code .`
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+In order to visualize the project in your browser either open the index.html file directly from the file explorer or consider using a [live server for development ](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
 
-## 🤝 Contributing
+## Authors
 
-Contributions, issues, and feature requests are welcome!
+👤 **Robin Benitez Mora**
 
-Feel free to check the [issues page](../../issues/).
+- GitHub: [@robinbenitezmora](https://github.com/robinbenitezmora)
+- Twitter: [@Mecanico_Robin](https://twitter.com/mecanico_robin)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/robin-benitez-mora-manizales)
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+-I thank Microverse for its documentation, and my mentor, David Verde for his support, guidance and patient by sharing his large knowledge.
```

Comment:
**First of all, I can say that you nailed the project. It's perfect. However, some places need to be improved.** 

- [ ] In your readme file, you need to give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) for his design. Kindly implement this change. Project requirement section about [giving credits...](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.)

Reply:
I am repeating the block and I have asked my classmates how to do the integration test and none of them know how to correct it because I get a negative result on the one I create. 
However, I have noticed that all my classmates from last week were approved for the project without those tests or they were given as optional.

As far as I can see I am going to have to lose the module, for something that I can improve over time, but that for you is totally indispensable. I simply don't know how to do it, nor can I find support... I will have to repeat the whole module again.